### ISSUE TITLE
Project is abandoned?

### ISSUE THREAD

**Issue by jackprogramsjp** [CSFrequency/react-firebase-hooks#310]
Seems like there hasn't been a release for almost a year and there are a lot of open GitHub issues that haven't been responded to.

**Comment by tylertaewook** [comment]
yeah, I'm not sure how consistent project update will be regarding future updates and fixes. Looking for more well-maintained projects at the moment

**Comment by jackprogramsjp** [comment]
I recommend [React Fire](https://github.com/FirebaseExtended/reactfire) as it is still being maintained and updated.

**Comment by Orick08** [comment]
Thanks for the recommendation, hope someday this project come back.

**Comment by steveoh** [comment]
reactfire also seems to be abandoned. Any other recommendations?

**Comment by 0x80** [comment]
You might be interested in this https://github.com/0x80/typed-firestore-react

There are other flavors for server and react-native
